src/main/webapp/app/admin/statistics/vm-statistics.service.ts

```diff
@@ -9,11 +9,11 @@ import { Injectable } from '@angular/core';
 export class VmStatisticsService {
     constructor(private http: HttpClient) {}
 
-    getVmStatistics(from: Date, to: Date): Observable<VmStatistics> {
-        return this.http.get<VmStatistics>(SERVER_API_URL + 'api/statistics/vm-usage/2010-01-01/2019-02-01');
+    getVmStatistics(from: Date, to: Date): Observable<VmStatistics[]> {
+        return this.http.get<VmStatistics[]>(SERVER_API_URL + 'api/statistics/vm-usage/2010-01-01/2019-02-01');
```

Comment:
Don't you feel like it should be configurable?

Reply:
jeez and its your code m8

Reply:
Its your PR

Reply:
My only error was approving your code but ok